ui/pages/groups/add.js

```diff
@@ -0,0 +1,185 @@
+import Head from 'next/head'
+import Link from 'next/link'
+import { useState, useRef } from 'react'
+import useSWR, { useSWRConfig } from 'swr'
+import { useRouter } from 'next/router'
+
+import ErrorMessage from '../../components/error-message'
+import Fullscreen from '../../components/layouts/fullscreen'
+import TypeaheadCombobox from '../../components/typeahead-combobox'
+
+function EmailsSelectInput({ selectedEmails, setSelectedEmails }) {
+  const { data: { items: users } = { items: [] } } = useSWR('/api/users')
+
+  const [query, setQuery] = useState('')
+  const inputRef = useRef(null)
+
+  const selectedEmailsId = selectedEmails.map(i => i.id)
+
+  const filteredEmail = [...users.map(u => ({ ...u, user: true }))]
+    .filter(s => s?.name?.toLowerCase()?.includes(query.toLowerCase()))
+    .filter(s => s.name !== 'connector')
+    .filter(s => !selectedEmailsId?.includes(s.id))
+
+  const removeSelectedEmail = email => {
```

Comment:
This one's personal preference but we do  `function removeSelectedEmail(email) {` elsewhere

Reply:
ugh i kept making this mistake - I am too used to the arrow function